electron/main.ts

```diff
@@ -54,13 +54,20 @@ import {
   weblateUrl
 } from './constants'
 import { handleProtocol } from './protocol'
-
+import { listenStdout } from './logger'
 const { showErrorBox, showMessageBox,showOpenDialog } = dialog
 const isWindows = platform() === 'win32'
 
 let mainWindow: BrowserWindow = null
 
 function createWindow(): BrowserWindow {
+  listenStdout().then((arr) => {
+    const str = arr.join('\n')
+    const date = new Date().toDateString()
+    const path = `${app.getPath('crashDumps')}/${date}`
```

Comment:
Where this path would be? its where Heroic is installed? Because then we could have permission issues. It might be good to save the logs under `$HOME/.config/heroic/` if that's the case.

Reply:
It is platform dependant. I can modify it using app.setPath("crashDumps",
value). But usually, it is under home.

Reply:
I noticed that we have a file under `.config/Heroic` called Crash Reports so it's probably that folder.

Reply:
Yeah!